### PR TITLE
Windows builds and small fixes

### PR DESCRIPTION
**Pull request by four0four**
- fixed crash when no config file is selected
- fixed functionality with non-UNIX dir paths
- normalized to UNIX line endings
- added windows vc15 configs
- added windows binaries

Just a few misc things :)

### CODE REVIEW

File: LoadProcConfig/LoadProcConfig.cpp [hunk 1]
```diff
@@ -1,179 +1,190 @@
...
+
+char device[MAXSTR] = "";
+char cfgfile[QMAXFILE];
+
+#ifdef _WIN32
+	char dir_sep = '\\';
+	char dir_up[] = "..\\";
+#else
+	char dir_sep = '/';
+	char dir_up[] "../";
```
Comment (alexhude) [inline LoadProcConfig/LoadProcConfig.cpp]: missed '='

**Review by alexhude — CHANGES_REQUESTED** [review]
Awesome PR! Thanks for line endings, no idea how this happened :)
Just fix this tiny bit and I will merge it.

**Comment by alexhude** [comment]
Merged! Thanks, mate.

**Comment by four0four** [comment]
You can tell what platforms I run IDA on...

No problem! Thanks for writing the tool I've been meaning to :)

**Pull request merged**